test/distributed/fsdp/test_fsdp_hybrid_shard.py

```diff
@@ -1,19 +1,17 @@
 # Owner(s): ["oncall: distributed"]
 
 import contextlib
-from functools import partial
-from collections import Counter
 import sys
+from collections import Counter
```

Comment:
Could we have formatting changes separated out and in a different PR? Formatting changes with logical changes make it harder for reviewers to understand the critical parts of the PR to review.

Reply:
Sorry about that. I separated it out. Eventually, we should try to have all developers use ufmt so that we can ufmt files right before pushing. Converging to the same formatter saves the uncertainty of how code should be formatted, and we chose ufmt since PyTorch recommends that.

Reply:
Curious about how we will encourage this practice. `lintrunner` is today's automated tool that can run pre-commit and is enforced by PyTorch CI. 

Should we work with dev infra / CI folks to add ufmt to `lintrunner`? Without automation, enforcement of a linting standard is prone to breaking.